.github/workflows/ci-custodian-rules.yaml

```diff
@@ -8,6 +8,7 @@ env:
   CLOUDSDK_CORE_PROJECT: ${{ secrets.GOOGLE_CLOUD_PROJECT }}
 jobs:
   remove-unattached-disks:
+    if: github.repository == 'kubeapps/kubeapps'
```

Comment:
Just curious, don't github actions have something like "this project" instead of having to write the project name?

Reply:
In this specific case it wouldn't work for us. When someone forks the repo, "this" would be the forked repo and thus the workflow would run 😄 
I got it from Github documentation [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-run-job-for-specific-repository).

Reply:
haha, you're right, of course (need more coffee to work). That's the purpose of the PR indeed, limiting forks 🤦